app/models/school.rb

```diff
@@ -140,4 +146,38 @@ def dfe_number
       establishment_number,
     ].join("/")
   end
+
+  def secondary_or_equivalent?
+    secondary_phase? || secondary_equivalent_special? || secondary_equivalent_alternative_provision?
+  end
+
+  private
+
+  def secure_unit?
+    school_type_group == "other" && school_type == "secure_unit"
```

Comment:
I would suggest that the system is more robust if we only check `school_type` here. It might be the case today that all secure units also happen to have a `school_type_group` of `"other"`, but there is nothing to guarantee that will be the case in the future.

```suggestion
    school_type == "secure_unit"
```